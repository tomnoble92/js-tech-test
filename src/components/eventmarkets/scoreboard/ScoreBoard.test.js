import React from 'react';
import { shallow } from 'enzyme';
import ScoreBoard from './Scoreboard'

describe('ScoreBoard should render', () => {

    it('should be render with given strings', () => {
        const teams = ['home', 'away']
        const scores = {
            'home': 1,
            'away': 6
        }
        const component = shallow(<ScoreBoard teams={teams} scores={scores}  />)
        expect(component).toMatchSnapshot()
    })
})