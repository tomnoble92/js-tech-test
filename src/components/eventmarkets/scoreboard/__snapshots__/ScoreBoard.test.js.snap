// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreBoard should render should be render with given strings 1`] = `ShallowWrapper {}`;
